src/button.scss

```diff
@@ -89,6 +105,11 @@ $block: #{$fd-namespace}-button;
   padding-left: $fd-button-compact-padding-x;
   padding-right: $fd-button-compact-padding-x;
   line-height: $fd-button-compact-line-height;
+
+  &::before {
+    top: -($fd-button-compact-additional-spacing + 0.0625rem);
+    bottom: -($fd-button-compact-additional-spacing + 0.0625rem);
```

Comment:
I think this 0.0625rem can be a variable as well :D

Reply:
Variable name has been changed :)